docs/configuration.md

```diff
@@ -35,6 +35,7 @@ Pulsar is a multi-tenant system that requires users to specify the [tenant and n
 | kafkaNamespace         | The default namespace of Kafka topics          | default |
 | kafkaMetadataTenant    | The tenant used for storing Kafka metadata topics    | public  |
 | kafkaMetadataNamespace | The namespace used for storing Kafka metadata topics | __kafka |
+| kopAllowedNamespaces   | The allowed namespace to list topics with a comma separator.<br>For example, "public/default,public/kafka".<br>If it's not set or empty, the allowed namespaces will be "<kafkaTenant>/<kafkaNamespace>". | |
```

Comment:
> If it's not set or empty, the allowed namespaces will be "<kafkaTenant>/<kafkaNamespace>"

Doesn't this change the default behavior? I thought if it is not set or empty, the allowed namespaces will be all namespaces, no?

Reply:
It changes the default behavior. It it's not set or empty, the allowed namespaces will only be the default namespace (`public/default`).

Reply:
The current semantic before this PR could go back to https://github.com/streamnative/kop/pull/206, which allows users to specific a topic of any namespace without any SASL authentication.
- Before #206, `AdminClient#listTopics()` returns the topic list of `<kafkaTenant>/<kafkaNamespace>`.
- Before this PR, `AdminClient#listTopics()` returns the topic list of all namespaces. However, this semantic is dangerous and may cause heavy load when KoP is enabled on an existing cluster with a lot of topics even if authentication is enabled.
- After this PR, `AdminClient#listTopics()` returns the topic list of `<kafkaTenant>/<kafkaNamespace>`. What's different is, we can configure the `kopAllowedNamespaces`. Then `AdminClient#listTopics()` can return the topic list of specific namespaces.

Reply:
@sijie Is there any more question?

Reply:
Okay. Make sense.